### ISSUE TITLE
line 217 should be vehicle.wake_up() instead of v.wake_up()

### ISSUE THREAD

**Issue by togonow** [therippa/tesla-bitbar#1]
current script didn't work until i updated line 217 to vehicle.wake_up() instead of v.wake_up()

```
    # loop through vehicles, print menu with relevant info       
    for i, vehicle in enumerate(c.vehicles):
        if prefix:
            print get_name(vehicle['display_name'])
        # print vehicle
        vehicle.wake_up()
```

**Comment by stuartjmoore** [comment]
Should be fixed by https://github.com/therippa/tesla-bitbar/pull/2

**Comment by therippa** [comment]
Fixed in #2
